cmd/import_data.go

```diff
@@ -111,8 +119,79 @@ func (cmd *ImportDataCmd) handleCsv(ctx context.Context, infoSchema *spanner.Inf
 
 }
 
+func (cmd *ImportDataCmd) handleDump(ctx context.Context, dbUri, dialect string, spannerAccessor *spanneraccessor.SpannerAccessorImpl) error {
+	// TODO: handle POSTGRESQL
+	driver := constants.MYSQLDUMP
+
+	// TODO: handle GCS
+	dumpReader, err := os.Open(cmd.sourceUri)
+	if err != nil {
+		if err != nil {
+			return fmt.Errorf(fmt.Sprintf("can't read dump file: %s due to: %v\n", cmd.sourceUri, err))
+		}
+	}
+
+	defer dumpReader.Close()
+
+	importDump := &import_data.ImportFromDumpImpl{
+		ProjectId:  cmd.project,
+		InstanceId: cmd.instanceId,
+		DbName:     cmd.dbName,
+		DumpUri:    cmd.sourceUri,
+		DumpReader: dumpReader,
+		Driver:     driver,
+	}
+
+	expressionVerificationAccessor := &expressions_api.ExpressionVerificationAccessorImpl{
+		SpannerAccessor: spannerAccessor,
+	}
+
+	processDump := &conversion.ProcessDumpByDialectImpl{
```

Comment:
is the test coverage for this method sufficient ?

Reply:
This interface expose only 1 function. The job of the function is to call the respective API for the specific source format. 

UT for this will be added in the subsequent PR.